bin/meta_xform.py

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env python
+
+"""
+Transform metadata .jsonl file from CKAN 2.1 alignment to 2.3
+    Usage::
+    python meta_xform.py <input file> <output file>
+"""
+
+from os import readlink
+
+import gzip
+import logging
+import os.path as path
+import shlex
+import subprocess
+import sys
+
+# from pylons.i18n import _
+import simplejson
+
+SUFFIX_FRA = '_fra'
+LEN_SUFFIX_FRA = len(SUFFIX_FRA)
+LANG_KEYS = ('en', 'fr')
+SP_SP = '  '
+SP_PIPE_SP = ' | '
+
+# new schema description, data field choices
+sd_new = None
+sd_new_dfc = None
+
+def _is_gzipped(fpath):
+    """
+    Return True if input path is a gzipped file, False otherwise
+    """
+    cmd = shlex.split('file --mime-type {0}'.format(fpath))
```

Comment:
this will fail if your file path contains spaces (and other special shell characters)

I think it's better to remove this `_is_gzipped` function and let the code fail in the gzip module instead. If you would like a friendlier error just catch the exception raised from gzip in the main function.